src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -84,6 +84,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     request.getSession().setAttribute("user", username);
+    if(username.equals("lriffle")) {
```

Comment:
This could be a static vector with just "lriffle" in it, then this code would ask if the username is in the vector. But that can be in a follow up.